WordPress/Classes/ViewRelated/Notifications/Views/NotificationDetailUserView.swift

```diff
@@ -0,0 +1,174 @@
+import SwiftUI
+import DesignSystem
+
+struct NotificationDetailUserView: View {
+	typealias FollowConfig = NotificationDetailUserView.FollowActionConfiguration
+
+    let config: Configuration
+
+    init(avatarURL: URL?, username: String?, blog: String?, onUserClicked: @escaping () -> Void) {
+        self.config = Configuration(avatarURL: avatarURL, username: username, blog: blog, onUserClicked: onUserClicked)
+    }
+
+    init(
+        avatarURL: URL?,
+        username: String?,
+        blog: String?,
+        isFollowed: Bool,
+        onUserClicked: @escaping () -> Void,
+        onFollowClicked: @escaping (Bool) -> Void
+    ) {
+        self.config = Configuration(
+            avatarURL: avatarURL,
+            username: username,
+            blog: blog,
+            followActionConfig: FollowConfig(isFollowed: isFollowed, onFollowClicked: onFollowClicked),
+            onUserClicked: onUserClicked
+        )
+    }
+
+    var body: some View {
+        HStack {
+            Button(action: config.onUserClicked) {
+                HStack(spacing: .DS.Padding.split) {
+                    AvatarsView(style: .single(config.avatarURL))
+                    VStack(alignment: .leading) {
+                        if let username = config.username {
+                            Text(username)
+                                .style(.bodySmall(.regular))
+                                .foregroundStyle(Color.DS.Foreground.primary)
+                                .lineLimit(1)
+                        }
+                        if let blog = config.blog {
+                            Text(blog)
+                                .style(.bodySmall(.regular))
+                                .foregroundStyle(Color.DS.Foreground.secondary)
+                                .lineLimit(1)
+                        }
+                    }
+                    Spacer()
+                }
+            }
```

Comment:
Nit: It would enhance readability to create smaller pieces in the form of 
`private var somethingButton: some View`
`private var somethingHStack: some View`

Reply:
Extracted some views here https://github.com/wordpress-mobile/WordPress-iOS/pull/23178/commits/d9779f64bb9779251f2f85339dae6fa83f9b10d7